src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,20 @@
+const createSrcset = (originCode, locator) => {
+  let imgSet = '';
+  const resolutions = [240, 320, 480, 624, 800];
+  if (originCode !== 'pips') {
+    const overridableOriginCode = originCode || 'cpsdevpb';
```

Comment:
The fallback value for originCode shouldn't be needed anymore. You should be able to just use originCode.

I note that there is the comment in the Image Container. https://github.com/bbc/simorgh/blob/latest/src/app/containers/Image/index.jsx#L25 we should be okay to remove this fallback now, since the work was done to the CMS to ensure we always have an originCode.

Reply:
@sareh Do you mean remove it in `getIchefURL` too?